src/test/java/com/larscheidschmitzhermes/nexus3/github/oauth/plugin/GithubApiClientTest.java

```diff
@@ -198,4 +198,18 @@ public void principalCacheHonorsTtl() throws Exception {
         Mockito.verifyNoMoreInteractions(mockClient);
     }
 
-}
\ No newline at end of file
+    @Test
+    public void shouldHaveBothOrgs() throws Exception {
+        HttpClient mockClient = fullyFunctionalMockClient();
+        config.setGithubOrg("TEST-ORG,TEST-ORG2");
+        GithubApiClient clientToTest = new GithubApiClient(mockClient, config);
+        GithubPrincipal authorizedPrincipal = clientToTest.authz("demo-user", "DUMMY".toCharArray());
+        MatcherAssert.assertThat(authorizedPrincipal.getRoles().iterator().next(), Is.is("TEST-ORG/admin"));
+
+        HttpClient mockClient2 = fullyFunctionalMockClient();
+        config.setGithubOrg("TEST-ORG2,TEST-ORG");
+        GithubApiClient clientToTest2 = new GithubApiClient(mockClient2, config);
+        GithubPrincipal authorizedPrincipal2 = clientToTest2.authz("demo-user", "DUMMY".toCharArray());
+        MatcherAssert.assertThat(authorizedPrincipal2.getRoles().iterator().next(), Is.is("TEST-ORG/admin"));
```

Comment:
this is exactly the same as in L207 - shouldn't we assert `TEST-ORG2/admin` here?

Reply:
Not in this case, because that would require to create and add a new org to the `mockTeams()` and `mockOrg()` should expect an array, not a string... this would require to modify a couple of other tests, and I didn't want to bloat this PR.

In that test, I just wanted to verify, given multiple orgs allowed, they are parsed as an array, and even if one org is attached to the user, it is still able to log in, regardless the position of the org in the list (so it is properly parsed into an array).

Should I make bigger changes to the test file?

Reply:
Aaah, now I see it! What about renaming the test to something like „shouldAcceptOrgAnywhereInList“ ?